src/searchReplace.tsx

```diff
@@ -44,13 +55,15 @@ export class SearchReplaceModel extends VDomModel {
       this._queryResults = data;
       this.stateChanged.emit();
       console.log(data);
+      this.isLoading = false;
```

Comment:
We should move this after the try-catch block to ensure it is turn off even if an error occurs.

Reply:
done